ColumnReinforcement/ColumnReinforcement.py

```diff
@@ -0,0 +1,371 @@
+                )
+            )
+            self.hideXdirRebarsWidget()
+            self.hideYdirRebarsWidget()
+
+    def number_radio_clicked(self):
+        """ This function enable number field and disable spacing field in UI
+        when number radio button is clicked."""
+        self.form.spacing.setEnabled(False)
+        self.form.number.setEnabled(True)
+
+    def spacing_radio_clicked(self):
+        """ This function enable spacing field and disable number field in UI
+        when spacing radio button is clicked."""
+        self.form.number.setEnabled(False)
+        self.form.spacing.setEnabled(True)
+
+    def getMainRebarType(self):
+        """ This function is used to find Main Rebars Type and update UI
+        accordingly."""
+        self.main_rebar_type = self.form.mainRebarType.currentText()
+        if self.main_rebar_type == "LShapeRebar":
+            self.form.mainRebarOrientationWidget.show()
+        else:
+            self.form.mainRebarOrientationWidget.hide()
+
+    def getXDirRebarType(self):
+        """ This function is used to find Rebar Type of rebars placed along
+        X-Direction and update UI accordingly."""
+        self.xdir_rebar_type = self.form.x_dirRebarType.currentText()
+        if self.xdir_rebar_type == "LShapeRebar":
+            self.form.x_dirRebarOrientationWidget.show()
+        else:
+            self.form.x_dirRebarOrientationWidget.hide()
+
+    def getYDirRebarType(self):
+        """ This function is used to find Rebar Type of rebars placed along
+        Y-Direction and update UI accordingly."""
+        self.ydir_rebar_type = self.form.y_dirRebarType.currentText()
+        if self.ydir_rebar_type == "LShapeRebar":
+            self.form.y_dirRebarOrientationWidget.show()
+        else:
+            self.form.y_dirRebarOrientationWidget.hide()
+
+    def getHookExtendAlong(self):
+        """ This function is used to find HookExtendAlong value from UI."""
+        self.main_hook_extend_along = self.form.mainRebarHookExtendAlong.currentText()
+        self.xdir_hook_extend_along = self.form.x_dirRebarHookExtendAlong.currentText()
+        self.ydir_hook_extend_along = self.form.y_dirRebarHookExtendAlong.currentText()
+
+    def hideXdirRebarsWidget(self):
+        """ This function hide widget related to Rebars placed along
+        X-Direction."""
+        self.form.x_dirRebarsWidget.hide()
+
+    def hideYdirRebarsWidget(self):
+        """ This function hide widget related to Rebars placed along
+        Y-Direction."""
+        self.form.y_dirRebarsWidget.hide()
+
+    def showXdirRebarsWidget(self):
+        """ This function show widget related to Rebars placed along
+        X-Direction."""
+        self.form.x_dirRebarsWidget.show()
+
+    def showYdirRebarsWidget(self):
+        """ This function show widget related to Rebars placed along
+        Y-Direction."""
+        self.form.y_dirRebarsWidget.show()
+
+    def gettupleOfNumberDiameter(self, diameter_string):
```

Comment:
This function should be present in `Rebarfunc.py` file because in future you will use this function everywhere. :)